docs/user-guide.md

```diff
@@ -0,0 +1,249 @@
+# User guide
+
+## Rules
+
+* Rules are standalone.
+* Rules are unopinionated, i.e:
+  * None of the rules are turned on by default.
+  * None of the rules have default values for options.
+
+### Rule names
+
+Rules are:
+
+* Made of lowercase words separated by hyphens.
+* Usually split into two parts:
+  * The first describes what [*thing*](http://apps.workflower.fi/vocabs/css/en) the rule applies to.
+  * The second describes what the rule is enforcing.
```

Comment:
Would it be more accurate to say this second part is what the rule is "checking"?

Because the "always" or "never" actually determines what gets "enforced" ...